ecommerce/products/management/commands/create_fixtures.py

```diff
@@ -0,0 +1,28 @@
+import random
+
+from django.core.management.base import BaseCommand
+from products.utils.fixtures import word_name, fake_description, fake_price,\
+    fake_date, \
+    choice_from_choice_class, fake_product, fake_product_item
+
+class Command(BaseCommand):
+    help = 'Generate random product'
+
+    def add_arguments(self, parser):
+            parser.add_argument('count', type=int, help='Indicates the number of products to be created')
+
+    def handle(self, *args, **kwargs):
+
+        count = kwargs['count']
+
+        for i in range(count):
+
+            product = fake_product()
+
+            print(product)
+            print(product.created_at)
```

Comment:
Django `BaseCommand` has an in-built method for printing which is better cos it is easier to test. replace `print` with `self.stdout.write`